dev-requirements.txt

```
@@ -2,5 +2,5 @@ black
 Django >= 2.2
 flake8
 mock >= 2.0.0
-python-ldap >= 3.0
+python-ldap >= 3.1
```

Shouldn’t be needed for this PR.
```suggestion
python-ldap >= 3.0
```

---

Yes, and I added `isort`, `mypy`, and `tox` to the dev dependencies as well, but after #228 has been merged, this would not be necessary any more.